build/package/send-slack-notification.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Copyright 2021 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -Eeou pipefail
+
+VERSION=$(git describe | cut -d "v" -f 2)
+
+curl --header "Api-User:${evergreen_user:?}" \
```

Comment:
[q] I wonder if evergreen doesn't have a go client for their API

Reply:
I have only found the python client https://github.com/evergreen-ci/evergreen.py